### ISSUE TITLE
controls:false option blocks the jwplayer

### ISSUE THREAD

**Issue by qpi** [jwplayer/jwplayer#1248]
### Expected Behavior

*The player shoud play media when the controls option is false*

### Actual Behavior

*The player do not play the mp4 (or maybe other types) when the control option is false*

### Steps to reproduce

1. open Your test site: http://demo.jwplayer.com/stream-tester/
2. open the console and run this code:

`
jwplayer("my-video").setup({
	analytics:{
		enabled: false
	},
	aspectratio:"16:9",
	autostart:false,
	controls:false,
	file:"http://distribution.bbb3d.renderfarming.net/video/mp4/bbb_sunflower_1080p_60fps_normal.mp4",
	key:"61Z3+bRe9a20fwsUdaUino/Tfcv5TcwLFsN4uw==",
	primary:"html5",
	type:"mp4",
	width:"100%"
});
`
3. You will see the play does not initialized and don't play the media, but if You remove the controls option, it works fine
4. In my context on my webpage where I use the jwplayer with the same configuration I saw this message:
`MediaError
code:4
MEDIA_ERR_SRC_NOT_SUPPORTED`
5. I don't understant what is the relation between the "not supported media" and the disabled controls :(

### Environment

- *PC*
- *Windows | 10*
- *JW Player 7.4.3 | url*

----------------------------------------------------------------------------------------

This template is provided as a guide to filing bugs and feature requests. Please replace 
all text *in italics* with details describing your issue. When submitting a feature request, 
"Steps to reproduce" can be substituted with a user story or list of acceptance criteria.

**Comment by johnBartos** [comment]
Thanks for the report! I'll take a look when I get the chance

**Comment by johnBartos** [comment]
@qpi What browser and browser version are you using?

**Comment by qpi** [comment]
I use Google Chrome	51.0.2704.84 on Windows 10 64bit

**Comment by arup87** [comment]
@qpi I could not reproduce the issue with Chrome 51.0.2704.84 on Windows 10 64bit. Playback starts when I call play from the api.

**Comment by qpi** [comment]
Sorry guys, I debugged my code deeply and the reason was not the controls:false, it was CORS problem. 
Before I pasted the sample code I changed the file url, because mine was internal, so that's why I could not send it for the bug.
Finally I added the crossorigin="use-credentials" argument for the <video> element what the jwplayer has created and after it was okay.
Will this "crossorigin" attribute supported by the jwplayer in the near future?
